ui/Screen/Settings.svelte

```diff
@@ -170,11 +170,11 @@
       <div class="section-item-single">
         <div class="info">
           <p class="typo-text-bold">
-            Seeds help you find more projects and people on the network.
+            Seeds help you find more projects and users on the network.
```

Comment:
I wonder if we need to think more about this based on this comment: https://github.com/radicle-dev/radicle-link/pull/436#issuecomment-732407411 @abbey-titcomb @cloudhead

Reply:
Yeah, although in the context of the app we can talk about users. It was changed in the protocol because protocols don't really have "users".

Reply:
Chatting with Kim about this now. Will open an issue if we need to change about it. But I think what @cloudhead captures it.